src/docs/product/alerts-notifications/issue-alerts.mdx

```diff
@@ -0,0 +1,65 @@
+| New event detection | Sentry detects a new event |
+| New event detection | Sentry detects a new event with tags matching {**key**} {**match**} {**value**}. A **key**: any tag. A **match**: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set. A **value**: any key’s value |
+| New event detection | Sentry detects a new event with {**attribute**} {**match**} {**value**}. An **attribute**: `message`, `platform`, `environment`, `type`, `exception.type`, `exception.value`, `user.id`, `user.email`, `user.username`, `user.ip_address`, `http.method`, `http.url`, `stacktrace.code`, `stacktrace.module`, or `stacktrace.filename`. A **match**: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set. A **value**: any attribute’s value. NOTE: When viewing an event, if an attribute is unclear or missing, you can view its JSON payload to determine its value. |
+| New event detection | Sentry detects a new event with level {**match**} {**level**}. A **match**: equal to, less than or equal to, greater than or equal to. A **level**: fatal, error, warning, info, debug, or sample. |
+
+## Actions
+
+Actions are the configurations that determine what will happen when an alert is triggered. The following actions are available:
+
+- Send an email to either [Issue Owners](/product/error-monitoring/issue-owners/), Team, or [Member](/accounts/membership/#member).
+- Send a notification to an [integration](/product/integrations/), which includes [PagerDuty](/product/integrations/integrations/pagerduty/), [Slack](/product/integrations/integrations/slack/), and [Microsoft Teams](/product/integrations/integrations/msteams/). If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
+
+## Rate Limit
+
+The rate limit determines how frequently an issue alerts. After an issue triggers an alert, Sentry won't check the conditions and won't execute the actions for that issue until the rate limit period passes. The limit is set to perform the action according to one of these intervals:
+
+- minutes: 5, 10, 30, 60
+- hours: 3, 12, 24
+- one week or 30 days
+
+For example, if an issue violates alert conditions multiple times in a 30-min period, but your frequency threshold is 30 minutes, you’ll only get one alert.
+
+![A dropdown that adjusts the frequency of an alert.](alert_frequency.png)
+
+## Environment
+
+Environment control allows you to specify which environment qualifies for your issue alert:
+
+- All Environments
+- Any of your defined (and not hidden) environments.
+
+If you select "All Environments", Sentry checks individually for each environment, rather than combined events across environments.
+
+![A dropdown allowing you to choose an environment to attach to an alert.](alert_environment.png)
```

Comment:
This needs to be updated (please grab a screen from the link to the builder I posted above)